test/Microsoft.AspNet.Razor.Runtime.Precompilation.Test/project.json

```diff
@@ -23,6 +23,9 @@
     },
     "frameworks": {
         "dnx451": {
+            "frameworkAssemblies": {
+                "System.Text.Encoding": "4.0.0.0"
```

Comment:
It's due to the Testing change @kichalla made. Does it not like the empty string as the version number?